packages/components/src/ellipsis-menu/index.js

```diff
@@ -66,6 +76,14 @@ EllipsisMenu.propTypes = {
 	 * A function returning `MenuTitle`/`MenuItem` components as a render prop. Arguments from Dropdown passed as function arguments.
 	 */
 	renderContent: PropTypes.func,
```

Comment:
Gosh, I wish we could change this to children 😢 

Not gonna demand a change here, but I'd be grateful if we supported children in a backward compatible way like we originally considered with the List component.

Reply:
This one is actually more justifiable as the `Dropdown` component it uses, uses the exact same format. It would be nice if the `Dropdown` component was converted first to make use of children.

Reply:
That makes sense 🙂